packages/core/src/hooks/useClickableProps/__stories__/useClickableProps.stories.tsx

```diff
@@ -1,17 +1,25 @@
-import React, { useCallback, useRef } from "react";
+import React, { useCallback, useRef, RefObject, MouseEvent, KeyboardEvent } from "react";
 import useClickableProps from "../useClickableProps";
 import "./useClickableProps.stories.scss";
 
+interface ClickableProps extends React.HTMLAttributes<HTMLElement> {
```

Comment:
No need to re-create the interface, we already use such interface in the hook :) We can just import it

Reply:
![image](https://github.com/user-attachments/assets/7c16564d-bd1c-49e4-a353-3650b05bddfd)
getting this error with the imported type. Now can I change that interface?

Reply:
@talkor in the interface `  tabIndex?: string | number;` defined as this. But im getting error because it has to be number only. Either I cast the tabindex explicitly or change the type interface. What to do ?

Reply:
@ahammadabdullah there seem to be an issue with the return type of the ref in useClickableProps which might be a breaking change, probably using a generic type will be best here but it's out of the scope. Let's close it for now, appreciate the effort! We have more issues to tackle if you'd like🙏🏼Thanks!